mesheryctl/internal/cli/root/pattern/error.go

```diff
@@ -14,14 +14,35 @@
 
 package pattern
 
-import "github.com/layer5io/meshkit/errors"
+import (
+	"fmt"
+	"strings"
+
+	"github.com/layer5io/meshkit/errors"
+)
 
 const (
+	ErrImportPatternCode          = "mesheryctl-1001"
+	ErrInValidSourceCode          = "mesheryctl-1002"
+	ErrOnboardPatternCode         = "mesheryctl-1003"
+	ErrPatternFoundCode           = "mesheryctl-1004"
+	ErrInvalidPatternNameOrIDCode = "mesheryctl-1005"
```

Comment:
What is the difference between code 1005 and 1039 ?
It would be good to have all error codes in ErrPattern<specific-error>Code  format for better readability?

Reply:
I don't think, there's any difference. //  @RipulHandoo

Reply:
Yes. I missed this one.